Source/WebCore/page/EventHandler.cpp

```diff
@@ -486,6 +486,9 @@ void EventHandler::nodeWillBeRemoved(Node& nodeToBeRemoved)
 {
     if (nodeToBeRemoved.containsIncludingShadowDOM(RefPtr { m_clickNode }.get()))
         m_clickNode = nullptr;
+
+    if (nodeToBeRemoved.containsIncludingShadowDOM(m_lastElementUnderMouse.get()))
```

Comment:
Why doesn’t this have the `RefPtr { }.get()` the way the similar code above does?

Reply:
We probably need to do that. Safer C++ check is gonna fail otherwise.